packages/discovery-provider/plugins/verified-uploads/src/handlers/users.js

```diff
@@ -69,16 +69,11 @@ export default async ({ user_id, blocknumber }) => {
             .catch(console.error)
 
           if (Object.keys(data).length === 0) {
-            // wait for identity to load
-            await new Promise((resolve) => {
-              setTimeout(resolve, 5000)
-            })
             throw new Error('social handles not in identity yet')
```

Comment:
won't we hit this exception most of the time now?
should we keep the delay, but remove the retries?

Reply:
yeah, i agree with that change, but I still think removing the 5s delay here will make us fail to capture new accounts that are coming in verified from the start

Reply:
i think the 5s delay only applies to the error case so all it does is tack on an additional 5 second delay on top of the exponential back off. comparing the logs w this delay added back w the logs above

```
retrying  { user_id: 38872 } at 2023-11-27T21:29:22.034Z
data  { data: {} } at 2023-11-27T21:29:22.224Z
retrying  { user_id: 38872 } at 2023-11-27T21:29:28.878Z
data  { data: {} } at 2023-11-27T21:29:29.012Z
retrying  { user_id: 38872 } at 2023-11-27T21:29:36.515Z
data  { data: {} } at 2023-11-27T21:29:36.645Z
retrying  { user_id: 38872 } at 2023-11-27T21:29:49.406Z
data  { data: {} } at 2023-11-27T21:29:49.539Z
```

if you'd prefer it not retry and succeed the first attempt, then maybe we can add a delay before the retry block 🤷 but the difference is negligible imo.

Reply:
ok i trust ya!

Reply:
sounds good to me!